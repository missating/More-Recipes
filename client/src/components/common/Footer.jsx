import React from 'react';

const Footer = () => (
  <footer>
    <hr />
    <div className="container bg-color">
      <div className="row">

        <div className="col-md-12">
          <div className="text-center">

            <h5 style={{ color: '#333' }}>
              Vanessa Ating &copy;&nbsp;
              {new Date().getFullYear()}
            </h5>

          </div>
        </div>
      </div>
    </div>

  </footer>
);

export default Footer;
