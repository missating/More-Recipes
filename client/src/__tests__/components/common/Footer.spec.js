import React from 'react';
import toJSON from 'enzyme-to-json';
import Footer from '../../../components/common/Footer';

describe('Footer component', () => {
  it('Should render', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.find('div').length).toEqual(4);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});

