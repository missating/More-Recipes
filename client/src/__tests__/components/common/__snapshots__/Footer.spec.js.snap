// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should render 1`] = `
<footer>
  <hr />
  <div
    className="container bg-color"
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          className="text-center"
        >
          <h5
            style={
              Object {
                "color": "#333",
              }
            }
          >
            Vanessa Ating © 
            2018
          </h5>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
